addons/im_livechat/static/src/embed/common/chat_window_patch.js

```diff
@@ -27,6 +27,9 @@ patch(ChatWindow.prototype, {
             this.props.chatWindow.show({ notifyState: this.thread?.state !== "open" });
         } else {
             await super.close();
+            if (this.thread?.isTransient) {
+                this.thread?.delete();
```

Comment:
nitpicking: the optional chaining is not needed as we know for a fact there is a thread and it is transient.